tools/ts-generator/example/generated.ts

```diff
@@ -1,290 +1,284 @@
 // Code generated by v/tools/ts-generator/rpcgen_js.py
```

Comment:
generated code

Reply:
checking in generated code makes PRs ALOT bigger then they need to be... what benefit do we get from having this checked in?

Reply:
In the previous PRs, someone said that good to see the generated code because can see the result and check the logic like write fns, read fns and implemented server, because check that functions on `types_gen_js.py ` could be hard.